gitleaks.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+# This is an example of what adding gitleaks to a pre-commit hook would look like.
+
+gitleaksEnabled=$(git config --bool hooks.gitleaks)
+cmd="gitleaks --repo-path=./ --verbose --redact --pretty"
+if [ $gitleaksEnabled = "true" ]; then
+    $cmd
+    if [ $? -eq 1 ]; then
+cat <<\EOF
+Error: gitleaks has detected sensitive information in your changes.
+If you know what you are doing you can disable this check using:
+    git config hooks.gitleaks false
```

Comment:
It's never mind and can be removed. Simpler to use `--no-verify` flag instead of changing git config.